index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const greetings = require('./lib/greet.js');
+
+//console.log(greetings.greet('maria'));
```

Comment:
This is nitpicky, but worth mentioning. Try and eliminate comments in your final submission for grading. (i did it all the time in 401 just to piss off scott. i really believe it helps think through our process, but is superfluous for submission)